admin_manual/configuration/server/occ_command.rst

```diff
@@ -1413,6 +1414,18 @@ Remove a certificate:
 
  sudo -u www-data php occ security:remove [certificate name]
 
+.. _ransomware_commands_label:
+
+Ransomware
```

Comment:
I'd call this "Ransomware Protection" here and also put a link to its doc page.

Reply:
Also we probably should mention here that this is an ownCloud Enterprise App which needs to be installed and enabled to be able to use these commands.

Reply:
Both are excellent points.